test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -632,11 +632,12 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 			})
 		})
 
-		When("realtime and high power consumption enabled", func() {
-			It("[test_id:50993][crit:high][vendor:cnf-qe@redhat.com][level:acceptance]should update kernel arguments and tuned accordingly", func() {
+		When("realtime and high power consumption enabled and perPodPowerManagent disabled", func() {
+			It("should update kernel arguments and tuned accordingly", func() {
```

Comment:
Request to retain the Test ID. These test id 's are required to match with test results with our Test case management system